src/chocolatey/infrastructure.app/services/NugetService.cs

```diff
@@ -268,6 +268,17 @@ public void pack_run(ChocolateyConfiguration config)
                 properties["version"] = config.Version;
             }
 
+            // Add any other properties passed to the pack command overriding any present.
+            foreach (var property in config.PackCommand.Properties)
+            {
+                this.Log().Debug(() => "{0} property '{1}': {2}".format_with(

```

Comment:
Debug, Verbose, or Trace here? I think Debug is fine.

Any concerns about sensitive data being exposed here? I will need to check if we log these in new.